calculator.rb

```diff
@@ -0,0 +1,174 @@
+# Calculator
+# Platinum level
+# By Alice Rhomieux
+
+puts "Hello, welcome to the calculator."
+
+# List of supported operations, structured as a hash.
+# Each array is a family of terms that will be accepted for each operation.
+# The operator symbol is always the last element of the class so it can be selected with index [-1]
+# Oops, this is a global variable
+$supported_operations = {
+	addition: ["addition", "add", "plus", "+"],
+	subtraction: ["subtraction", "subtract", "minus", "sub", "-"],
+	multiplication: ["multiplication", "multiply", "times", "*"],
+	division: ["division", "divide", "div", "by", "/"],
+	exponent: ["exponent", "exp", "power of", "power", "**", "^"],
+	modulus: ["modulus", "modulo", "mod", "remainder", "%"]
+}
+
+# Check a number inputted by the user as a string and see if it has a decimal
+# If it has a decimal, convert it to a float, otherwise, convert it to an integer
```

Comment:
Good point. I could have the user pick which language or country they were operating from, and have different input handling logic to cover different number formatting.